controllers/metallb_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *MetalLBReconciler) reconcileResource(ctx context.Context, req ctrl.Requ
 
 func (r *MetalLBReconciler) SetupWithManager(mgr ctrl.Manager, bgpType string) error {
 	if bgpType == "" {
-		bgpType = bgpNative
+		bgpType = bgpFrr
```

Comment:
is this required?

Reply:
nope can remove it

Reply:
Code wise, we must have very strong reasons to drift from upstream.
We must put as much as we can in manifests.

Reply:
Reverted. This env var shouldn't be empty anyways (assuming the user doesn't touch the manifests)